### DISCUSSION TITLE
Issue Label Categories

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#580] [General]
We need a more robust system for issue labels. Here's a modified version of what I use on [pyarr](https://github.com/totaldebug/pyarr) that we can adopt in all idom-team repos.

- type/feature
- type/enhancement
- type/bug
- type/ci
- type/docs
- type/language
- type/question
- state/investigation
- state/paused
- state/WIP
- state/review needed
- state/wontfix
- state/blocked
- priority/critical
- priority/high
- priority/medium
- priority/low
- flag/help wanted
- flag/breaking changes

It looks like I don't have permission to change these settings on any of the repos so I'll need to rely on @rmorshea for this one.

**Comment by rmorshea** [comment]
Things have been somewhat improved: https://github.com/idom-team/idom/labels

Will add state labels if we need them.

**Reply by Archmonger** [reply]
Looks good to me. Could we get that duplicated over at `django-idom`? I can handle the issue relabeling over there to save you some time if you'd like.
